src/Core/Reflect.idr

```diff
@@ -68,9 +69,10 @@ reflectionttimp : String -> Name
 reflectionttimp n = NS reflectionTTImpNS (UN $ Basic n)
 
 export
-cantReify : NF vars -> String -> Core a
-cantReify val ty
-    = throw (GenericMsg (getLoc val) ("Can't reify as " ++ ty))
+cantReify : Ref Ctxt Defs => {vars : _} -> Env Term vars => NF vars -> String -> Core a
+cantReify val ty = do
+  logNF "reflection.reify" 10 "Can't reify as \{ty}" %search val
+  throw (GenericMsg (getLoc val) ("Can't reify as " ++ ty))
```

Comment:
```suggestion
cantReify : Ref Ctxt Defs => {vars : _} -> NF vars -> String -> Core a
cantReify val ty = do
  logNF "reflection.reify" 10 "Can't reify as \{ty}" (mkEnv emptyFC _) val
  throw (GenericMsg (getLoc val) ("Can't reify as " ++ ty))
```

Reply:
Oh, I got lost in types, `vars` parameter determines everything. Thank you!